src/engraving/types/typesconv.cpp

```diff
@@ -0,0 +1,256 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "typesconv.h"
+
+#include "log.h"
+
+using namespace mu::engraving;
+
+template<typename T>
+struct Item
+{
+    T type;
+    QString xml;
+    QString userName;
```

Comment:
Currently, the username _is_ translatable (because of `Q_TRANSLATE_NOOP`), but it is never really translated, as far as I can see. I could imagine that it is sometimes desired to have the untranslated username, and sometimes the translated version.

So you could change the userName member to `const char*`, and then add an additional function below, like `TConv::toTranslatedUserName`, which translates the username using `mu::qtrc` and returns a QString.

Alternatively, if you want to keep the translating outside of this module, the `TConv::toUserName` function should return `const char*`, since `mu::qtrc` needs that, and converting from QString to `const char*` every time is not so nice.

Reply:
You are right, I didn’t think about it.
I will change `userName` to `const char*`, and add `qtrc` to `toUserName` method.

Reply:
added qtrc